tasks/schemas/taskPlan.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "/tasks/taskPlan",
+  "title": "Task Plan",
+  "description": "An assignment of the task to an agent",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "integer",
+      "description": "Id referencing task plan"
+    },
+    "parentTaskPlanID": {
+      "type": "integer",
```

Comment:
can this be a `$ref` like `taskRecipeId`?

Reply:
Oh good catch